AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Commands/AWSSDKSwiftCLI/Subcommands/PrepareRelease.swift

```diff
@@ -235,13 +235,16 @@ struct PrepareRelease {
         previousVersion: Version
     ) throws {
         let commits = try Process.git.listOfCommitsBetween("HEAD", "\(previousVersion)")
-        
+        let featuresReader = FeaturesReader()
+
```

Comment:
Features reader is used to read the build message & mapping, then the read data is passed into the release notes builder.